src/test/java/org/fundacionjala/coding/christian/CesarCipherTest.java

```diff
@@ -0,0 +1,241 @@
+package org.fundacionjala.coding.christian;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Performed by christian.
+ */
+public class CesarCipherTest {
+
+    /**
+     *
+     */
+    private Cipher cesar;
+
+    private CipherVigenere vigenere;
+
+    private static final String VIGENERE_KEY = "LIMON";
+
+    /**
+     *
+     */
+    @Before
+    public void setUp() {
+        cesar = new Cipher();
+        vigenere = new CipherVigenere();
```

Comment:
This is wrong
because you are not using the two instances in all the tests

Consider splitting in two unit test class